### DISCUSSION TITLE
Documentation

### DISCUSSION THREAD

**Discussion by joaoceron** [marcusolsson/grafana-treemap-panel#29] [Ideas]
It would be nice to provide expected values. For example, a SQL table that shows the fields. It says that is expected "label" and "count". However, debugging why this is not working is not easy. For example, the following table could be printed. 

┌─frequency─┬─label ─┐
 │     10492           │ X                │
 │      8297            │ Y                │
 │      7460            │ X                │
└─────  ─┴─   ───┘

**Comment by marcusolsson** [comment]
Thanks for your feedback! If possible, I'd prefer not to give examples for each data source. The plugin is designed to be used for any data source so ideally I'd like to make it easier to understand from the "panel wizard".

What are you seeing? The table looks good to me. My bet would be that the frequency field is of type string rather than number.
